wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -106,6 +115,12 @@ def test_cleaned_fields(self):
         self.assertIsInstance(latest_revision_created_at, datetime)
         self.assertEqual(wp_link, "")
 
+    def test_no_title_faked(self):
+        wordpress_item = WordpressItem(self.no_title, self.logger)
+        title = wordpress_item.cleaned_data["title"]
+
+        self.assertTrue(title.startswith("no-title-available"))
```

Comment:
**Suggestion**: We may as well assert that the title is formatted with the post ID, too.

```suggestion
        self.assertEqual(self.no_title["wp:post_id"], "1000")  # just a fixture regression test
        self.assertEqual(title, f"no-title-available-1000")
```